commit 7948aeb71f539aedd0a1adf114ddc03dd6a19cfd (refs/changes/10/2210/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-07-12T11:27:39-07:00 (2 years, 3 months ago)
    
    TOOLS-1807 node updates for july 2017 node.js security releases (configure all node builds --without-snapshot)

diff --git a/nodeconfigs.json b/nodeconfigs.json
index a593b54..016a7f4 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -1,9 +1,36 @@
 [
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "version": "v4.8.4",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "gcc_version_prefix": "4.9.",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "build_tag": "gz"
+    },
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v4.8.4",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v4.8.4",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "build_tag": "zone64"
+    },
+
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "version": "v4.8.1",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "gcc_version_prefix": "4.9.",
         "rpath": "$ORIGIN/../lib",
         "include_gcc_runtime_libs": true,
@@ -14,7 +41,7 @@
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v4.8.1",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -22,7 +49,7 @@
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v4.8.1",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
 
@@ -30,7 +57,7 @@
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "version": "v4.6.1",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "gcc_version_prefix": "4.9.",
         "rpath": "$ORIGIN/../lib",
         "include_gcc_runtime_libs": true,
@@ -41,7 +68,7 @@
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v4.6.1",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
 
@@ -49,7 +76,7 @@
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "version": "v4.6.0",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "gcc_version_prefix": "4.9.",
         "rpath": "$ORIGIN/../lib",
         "include_gcc_runtime_libs": true,
@@ -60,7 +87,7 @@
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v4.6.0",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
 
@@ -69,7 +96,7 @@
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.12.17",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -77,7 +104,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.12.17",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -85,7 +112,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.12.17",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -93,7 +120,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.12.17",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -102,8 +129,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.12.17",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
     {
@@ -111,7 +138,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.12.17",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -121,7 +148,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.12.14",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -129,7 +156,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.12.14",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -137,7 +164,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.12.14",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -146,8 +173,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.12.14",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
     {
@@ -155,7 +182,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.12.14",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -165,7 +192,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.12.9",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -173,7 +200,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.12.9",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -181,7 +208,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.12.9",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -190,8 +217,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.12.9",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
     {
@@ -199,7 +226,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.12.9",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -209,7 +236,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.48",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -217,7 +244,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.48",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -225,7 +252,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.48",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -234,8 +261,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.10.48",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
     {
@@ -243,7 +270,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.48",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -255,7 +282,7 @@
         "patch_files": [
             "patches/ecdh-ciphers-support-v0.10.patch"
         ],
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zoneecdh"
     },
@@ -264,7 +291,7 @@
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.48",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
 
@@ -273,7 +300,7 @@
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.43",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
 
@@ -282,7 +309,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.42",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -290,7 +317,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.42",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -298,7 +325,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.42",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -307,8 +334,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.10.42",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
     {
@@ -316,7 +343,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.42",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -328,7 +355,7 @@
         "patch_files": [
             "patches/ecdh-ciphers-support-v0.10.patch"
         ],
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zoneecdh"
     },
@@ -338,7 +365,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.40",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -346,7 +373,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.40",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -354,7 +381,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.40",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -363,8 +390,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.10.40",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
     {
@@ -372,7 +399,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.40",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -384,7 +411,7 @@
         "patch_files": [
             "patches/ecdh-ciphers-support-v0.10.patch"
         ],
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zoneecdh"
     },
@@ -394,7 +421,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.32",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -403,7 +430,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.32",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -413,7 +440,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.30",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -421,7 +448,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.30",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -429,7 +456,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.30",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -438,8 +465,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.10.30",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
 
@@ -448,7 +475,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.29",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -456,7 +483,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.29",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -464,7 +491,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.29",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -473,7 +500,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.29",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -482,8 +509,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.10.29",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
 
@@ -492,7 +519,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.26",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
     {
@@ -500,7 +527,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.26",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -508,8 +535,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.10.26",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
     {
@@ -517,7 +544,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.26",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -527,7 +554,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.25",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
     {
@@ -535,7 +562,7 @@
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.25",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -544,7 +571,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.25",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -553,8 +580,8 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v0.10.25",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
 
@@ -563,7 +590,7 @@
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
         "version": "v0.10.24",
-        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64",
         "// users": "latest"
     },
@@ -572,7 +599,7 @@
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
         "version": "v0.10.24",
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
@@ -586,8 +613,8 @@
             "patches/node-5feedc5-socket-getFD.patch",
             "patches/node-reverse-2c41a80-crypto-check-key-type.patch"
         ],
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz",
         "// users": "sdc-amon"
     },
@@ -601,7 +628,7 @@
             "patches/node-5feedc5-socket-getFD.patch",
             "patches/node-reverse-2c41a80-crypto-check-key-type.patch"
         ],
-        "configure_flags": "--dest-cpu=ia32 --with-dtrace --openssl-use-sys --openssl-includes=/opt/local/include --openssl-libpath=/opt/local/lib --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --openssl-use-sys --openssl-includes=/opt/local/include --openssl-libpath=/opt/local/lib --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone",
         "// users": "manta-mako"
